<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popupWindow</class>
 <widget class="QDialog" name="popupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>0.800000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">Background-color: rgb(200,0,0);</string>
  </property>
  <widget class="QLabel" name="message">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>2000</width>
     <height>2000</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>75</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>You've been browing X application for Y minutes!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">Background-color: rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
